packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -168,18 +168,24 @@ export async function summarizeNow(summarizer: ISummarizer, reason: string = "en
 	const result = summarizer.summarizeOnDemand({ reason });
 
 	const submitResult = await timeoutAwait(result.summarySubmitted);
-	assert(submitResult.success, "on-demand summary should submit");
+	if (!submitResult.success) {
+		throw submitResult.error;
```

Comment:
Made this change because on summary failures, the error was not propagated. Instead, the assert message was sent up.

Reply:
yay, nice.